docs/outlook/spam-reporting.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Implement an integrated spam-reporting add-in (preview)
+description: Learn how to implement an integrated spam-reporting add-in in Outlook.
+ms.date: 07/14/2023
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Implement an integrated spam-reporting add-in (preview)
+
+With the number of unsolicited emails on the rise, security is at the forefront of add-in usage. Currently, partner spam-reporting add-ins are added to the Outlook ribbon, but they usually appear towards the end of the ribbon or in the overflow menu. This makes it harder for users to locate the add-in to report unsolicited emails. In addition to configuring how messages are processed when they're reported, developers also need to complete additional tasks to show processing dialogs or supplemental information to the user.
+
+The integrated spam reporting feature eases the task of developing individual add-in components from scratch. More importantly, it displays your add-in in a prominent spot on the Outlook ribbon, making it easier for users to locate it and report spam messages. Implement this feature in your add-in to:
+
+- Improve how unsolicited messages are tracked.
+- Provide better guidance to users on how to report suspicious messages.
+- Enable an organization's security operations center (SOC) or IT administrators to easily perform spam and phishing simulations for educational purposes.
+
+> [!IMPORTANT]
+> Features in preview shouldn't be used in production add-ins. We invite you to try out this feature in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+
+## Preview the integrated spam reporting feature
+
+To preview the integrated spam reporting feature in Outlook on Windows, you must install Version 2307 (Build 16626.10000) or later. Then, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/Windows) and select the **Beta Channel** option to access Office beta builds.
```

Comment:
I wonder if we should call out that it's only in **Beta Channel** and NOT in **Preview Channel** I actually don't know how far we want to push this, so we have to confirm with Juan. 

If you think you're already being explicit enough, then you can ignore this suggestion.

Reply:
Just to update we DO want to get this out to Preview Channel. But it is NOT enabled in Preview Channel right now. (so doc is fine as is, but there will be an update sometime in the future.